src/measure_theory/integral/set_integral.lean

```diff
@@ -787,6 +787,105 @@ end
 
 end tendsto_mono
 
+/-! ### First moment method -/
+
+section first_moment
+variables {μ : measure α} {s N : set α} {f : α → ℝ} {r : ℝ}
+
+/-- **First moment method**. An integrable function is smaller than its mean on a set of positive
+measure. -/
+lemma measure_le_set_integral_pos (hμ : μ s ≠ 0) (hμ₁ : μ s ≠ ⊤) (hf : integrable_on f s μ)
```

Comment:
Why do we need `μ s ≠ ⊤`?

Reply:
Also can you use the `\infty` notation?

Reply:
If `s` has infinite measure, then `(μ s).to_real = 0` and `∫ a in s, f a ∂μ / (μ s).to_real = 0`. So the result is false.